### ISSUE TITLE
/typography/links/link_colors/ Example for .scss is in .sass

### ISSUE THREAD

**Issue by KatieK2** [Compass/compass#1568]
On Page: http://compass-style.org/reference/compass/typography/links/link_colors/

I was looking for:
Example code in .scss syntax.

But instead I found:

<pre>a
  +link-colors(#00c, #0cc, #c0c, #ccc, #cc0)`</pre>

Is the documentation open-sourced?

**Comment by chriseppstein** [comment]
Yes. the docs are here: https://github.com/Compass/compass/tree/stable/compass-style.org

This actually comes from documentation in [a code comment](https://github.com/Compass/compass/blob/stable/core/stylesheets/compass/typography/links/_link-colors.scss#L12-L13).  It would be great if we converted all code comment documentation to be in scss syntax. Wanna do this?

**Comment by KatieK2** [comment]
I can have a crack at it some time. The nanoc build process looks a little hairier than I have time for right now, so it might be a bit of time.
